tests/integration/testsuites/istio-jwt/features/istio_jwt.feature

```diff
@@ -44,6 +44,22 @@ Feature: Exposing endpoints with Istio JWT authorization strategy
     And JwtAndUnrestricted: Calling the "/json" endpoint without token should result in status between 200 and 299
     And JwtAndUnrestricted: Teardown httpbin service
 
+  Scenario: Exposing a httpbin endpoint with JWT and no_auth in the same rule
```

Comment:
Maybe we can make this more expressive, so it's easier to understand what general behaviour is tested.
```suggestion
  Scenario: The APIRule is in status Error when there are handlers in the same rule that must not be combined with others
```